Objects/clinic/unicodeobject.c.h

```diff
@@ -582,7 +582,7 @@ PyDoc_STRVAR(unicode_strip__doc__,
 "strip($self, chars=None, /)\n"
 "--\n"
 "\n"
-"Return a copy of the string with leading and trailing whitespace remove.\n"
+"Return a copy of the string with leading and trailing whitespace removed.\n"
```

Comment:
This is a generated file.  Can you instead make the correction in `Objects/unicodeobject.c` and then run `make clinic`?

Reply:
@zware:
>  Can you instead make the correction in Objects/unicodeobject.c and then run make clinic?

Ah, I wasn't aware that Argument Clinic could not be ran against the header file directly, but that would make sense. I've personally never had to use it before, I've only briefly read over the documentation to try to help others.

Also, I was not aware of the existence of ``make clinic``, I thought the only option was to explicitly use ``./Tools/clinic/clinic.py <path>``.  It's not mentioned within the devguide as far as I can tell: https://devguide.python.org/search/?q=make+clinic&check_keywords=yes&area=default.

Should we perhaps mention it somewhere? Using ``make clinic`` is far more straightforward than using clinic.py on each of the C files that need it.

Reply:
Hmm, it looks like we also don't mention `make regen-all` anywhere in the devguide, which is probably the better target to remember since it includes `make clinic`.  On the other hand neither is available on Windows, so we should probably also mention the clinic script directly.